l10n_es_aeat_mod190/views/partner_view.xml

```diff
@@ -4,96 +4,123 @@
         <field name="name">res.partner.mod190.form.inherit</field>
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="base.view_partner_form" />
-        <field name="groups_id" eval="[(4, ref('l10n_es_aeat.group_account_aeat'))]" />
         <field name="arch" type="xml">
             <field name="ref" position="after">
                 <field name="incluir_190" />
             </field>
-            <page name="sales_purchases" position="after">
-                <page
-                    string="Modelo 190"
-                    name="mod190"
+        </field>
+    </record>
+
+    <record id="view_partner_form_inherit" model="ir.ui.view">
```

Comment:
Why another view? Inherit everything directly from `l10n_es_aeat.view_partner_form`

Reply:
Hello @pedrobaeza 
Because it depends on the **"incluir_190"** field the view will show when it is true and the **"incluir_190"** field depends on the **"ref"** field from the base.
So both views are made separately.

Reply:
As said, no need of 2 views. Put everything on the second one.